test/nerf/points3D.txt

```diff
@@ -0,0 +1,7338 @@
+# 3D point list with one line of data per point:
```

Comment:
I don't know the status of this feature/pr, but this file should live in https://github.com/kornia/data/tree/main/nerf (related to https://github.com/kornia/data/pull/3)